utils.py

```diff
@@ -0,0 +1,24 @@
+from enum import Enum
+import httpx
```

Comment:
Use the requests package instead. It is way easier to use and more the standard rather then httpx.

Reply:
I used the httpx package as we can make asynchronous api request with it. whereas request package only supports synchronous request

Reply:
So even with Requests package you can do async calls see: https://superfastpython.com/python-async-requests/

That is the way I typically handle those problems, but if you prefer to use HTTPX then that is okay too. Happy to keep it this way 👍